build.sbt

```diff
@@ -102,11 +102,12 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
     ,"uk\\.gov\\.hmrc\\.customs\\.notification\\.domain\\..*"
     ,".*(AuthService|BuildInfo|Routes).*"
   ).mkString(";"),
-  coverageMinimum := 99,
+  coverageMinimum := 98,
```

Comment:
why did we need to reduce the coverage? not saying whether it is right or wrong, want to know the reason it went down.